src/doc_builder/commands/convert_doc_file.py

```diff
@@ -68,8 +68,14 @@ def convert_command(args):
     with open(source_file, "r", encoding="utf-8") as f:
         text = f.read()
 
-    page_info = {"package_name": package_name, "page": source_file.with_suffix(".html").relative_to(doc_folder)}
+    page_info = {
+        "package_name": package_name,
+        "page": source_file.with_suffix(".html").relative_to(doc_folder),
+        "no_prefix": True,
```

Comment:
We don't want the whole prefix with master and language when resolving links to other doc pages or references in the source files.